_summaries/he2019momentum_2.md

```diff
@@ -3,17 +3,100 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: zayne-sprague
+score: 10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## What's the big idea
+
+A large dynamic dictionary of keys in a queue, built via a momentum encoder, can improve contrastive learning making unsupervised pretraining in computer vision viable and better than supervised pretraining.
+
+## What's hard about unsupervised pre-training in vision
+
+- Pretraining on images is difficult because they are in a continuous space, and the representations learned may not transfer
+
+- Some work has been done on pretraining computer vision models using pretext tasks, but by in large have failed to outperform supervised pretraining
```

Comment:
The paper gives a good comparison between NLP tasks and CV for pretraining and representation. I think including that comparison might help readers understand the context of the problem at hand more.

Reply:
Good point, I added a bit more to this section, mostly introducing why its easy in NLP vs why its hard in CV